p2p/__init__.py

```diff
@@ -809,11 +814,6 @@ def push_into_collection(self, code, content_item_slugs):
         """
         Push a list of content item slugs onto the top of a collection
         """
-        # Enforce that a list of slugs is passed in (not a string)
```

Comment:
It's a double check in case the clients that use the wrapper (like us) have incorrectly formatted their data (like us)